pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1,21 +1,21 @@
 # Before adding a new extension, read ./README.md
 
-{
-  autoPatchelfHook,
-  callPackage,
-  config,
-  fetchurl,
-  jdk,
-  jq,
-  lib,
-  llvmPackages,
-  llvmPackages_14,
-  moreutils,
-  protobuf,
-  python3Packages,
-  stdenv,
-  vscode-utils,
-  zlib,
+{ autoPatchelfHook
+, callPackage
+, config
+, fetchurl
+, jdk
+, jq
+, lib
+, llvmPackages
+, llvmPackages_14
+, moreutils
+, protobuf
+, python3Packages
+, stdenv
+, vscode-utils
+, zlib
+,
```

```suggestion
{ autoPatchelfHook
, callPackage
, config
, fetchurl
, jdk
, jq
, lib
, llvmPackages
, llvmPackages_14
, moreutils
, protobuf
, python3Packages
, stdenv
, vscode-utils
, zlib
,
```

Certify you have used [nixfmt-rfc-style](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=nixfmt) as formatter. It's only available in `unstable`.